frontend/src/_data/series.js

```diff
@@ -0,0 +1,20 @@
+/*!
+ * sri sri guru gaurangau jayatah
+ */
+
+const { readItems } = require('@directus/sdk');
+
+/**
+ * @param {EleventyGlobalData} data
+ */
+
+module.exports = ({ directus }) =>
+  directus.request(
+    readItems('series', {
+      fields: ['*.*.*'],
+      filter: { status: { _eq: 'published' } },
+      // Also, we assume that the `QUERY_LIMIT_MAX` env variable is not set on the server.
+      // Fetching only few items in the development ymode to speed up rebuilding the pages
```

Comment:
This line regarding the development mode is not relevant anymore.

Reply:
removed